src/common/templates/_databaseconnections.tpl

```diff
@@ -0,0 +1,93 @@
+{{/* Generates mongo environment variables
+{{ include "harnesscommon.dbconnection.mongEnv" . | nident 10 }}
+*/}}
+{{- define "harnesscommon.dbconnection.mongoEnv" }}
+{{- $type := "mongo" }}
+{{- $installed := (pluck $type .Values.global.database | first ).installed }}
+{{- $passwordSecret := (pluck $type .Values.global.database | first ).secretName }}
+{{- $passwordKey := (pluck $type .Values.global.database | first).passwordKey }}
+{{- $userKey := (pluck $type .Values.global.database | first).userKey }}
+{{- if $installed }}
+{{- include "harnesscommon.dbconnection.dbenvuser" (dict "type" $type "secret" "harness-secrets" "userKey" "mongodbUsername") }}
```

Comment:
shall we hardcode "harness-secrets" ? I think we don't need $installed `if` condition
the default global values could look like
```yaml
global:
  database:
    mongo:
      secretName: harness-secrets
      userKey: mongodbUsername
      passwordKey: mongodb-root-password
```

Reply:
I wanted to do this however the username and password are in different secrets